nix/nginz.nix

```
@@ -0,0 +1,105 @@
+{ nginxModules
+, pkg-config
+, callPackage
+, stdenv
+, symlinkJoin
+, dockerTools
+, writeTextDir
+, runCommand
+, gnugrep
+, coreutils
+, nginx
+, inotify-tools
+, dumb-init
+, cacert
+, bashInteractive
+, openssl
+, lib
+, makeWrapper
+, writers
+}:
+let
+  zauth = callPackage ./pkgs/zauth { };
+
+  nginzModules = nginxModules // {
+    zauth = {
+      src = ../services/nginz/third_party/nginx-zauth-module;
+      inputs = [ pkg-config zauth ];
+    };
+  };
+
+  nginz = nginx.override {
+    # openssl = (openssl.overrideAttrs (oldAttrs: {
+    #   patches = oldAttrs.patches or [] ++ [
+    #     ./patch.patch
+    #   ];
+    # }));
```

```suggestion
    openssl = openssl.override { openssl = openssl_1_1; };
```

(as per https://xeiaso.net/blog/nixos-nginx-openssl-1.x), it looks like we should use this for the time being.